README.md

```diff
@@ -33,15 +33,18 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
-
+    Semantic HTML are tags that contain some sort of meaning. 
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
+   - display block presents everything in a block and holds no html elements next to it.
+   - display inline has all the elements display in the current block on the same line.
 
 3. What are the 4 areas of the box model?
+    - conten, padding, border, margin
 
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
-
+    - this will align all the items on the center of the cross axis going horizontal.
```

Comment:
in the Flexbox mode the cross axis is vertical, the Main Axis is the one that goes horizontal